### ISSUE TITLE
Edit an action recorded by the recorder

### ISSUE THREAD

**Issue by liciavale** [elastic/synthetics-recorder#132]
As a user of the script recorder

- [ ] I want to be able to edit an action after that the script recorder automatically recorded it

So that I can optimize the command based on the needs of my flows

<details>
<summary>Deliverables</summary>

- [Figma file ]
- [Prototype ]

**Comment by justinkambic** [comment]
Note: as an implementation detail, we might need some additional investigation to find the types of actions we can show here, I'm thinking there might be a field mismatch on what we display in some cases.

**Comment by liciavale** [comment]
@justinkambic good point, I'll adapt the design to a more complex scenario to see how we can handle this

**Comment by justinkambic** [comment]
Resolved by https://github.com/elastic/synthetics-recorder/pull/137.
